sample_files/.travis.yml

```diff
@@ -9,6 +9,10 @@ python:
   - "2.7.13"
 
 addons:
+  # By default postgresql-9.1 is installed but there is issue related:
+  #  https://github.com/OCA/maintainer-quality-tools/issues/432#issuecomment-281580935
+  # Better use higher PostgreSQL version
+  postgresql: "9.4"
```

Comment:
why not 9.5?

Reply:
9.5 is fine too and It is the official version for `Ubuntu-16.04` then changed.